MetaMask/metamask-mobile
# chore: auto generate changelist

## **Description**

This PR creates a yarn command that does two things:
1.  creates csv file on the change diff between commits and categorizes by change type based off of the auto-changelog format
2. updates the changelog with the diff categorized by Added, Change, Fixed, Ops, Unknown 

## **Related issues**

Fixes: 

## **Manual testing steps**

1. Pull the branch down
2. Run `yarn setup`
3. Run `yarn changeset-documentation <base branch/tag/hash> <target branch/tag/hash>`
4. Observe output of `commits.csv` at root level and modified `changelog.md` file

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
